tests/FSharp.Core.UnitTests/SurfaceArea.fs

```diff
@@ -9,2511 +9,2515 @@ type SurfaceAreaTest() =
     [<Fact>]
     member this.VerifyArea() =
         let expected = @"
-Microsoft.FSharp.Collections.Array2DModule: Int32 Base1[T](T[,])
```

Comment:
I wonder what caused the re-ordering of the baseline output.  Oh shit it was the recent hashing PR, I bet!!!!!

Reply:
@nojaf  : I assume you just copy-pasted 'actual' into this expected string, right?

Can you please in ~\tests\FSharp.Core.UnitTests\LibraryTestFx.fs  improve the 'types' array to be sorted by typename?
This is what the 'expected' list looks like to be. And leave the sort extension in the code for others.

And then please re-paste the actual into expected using this change, this should eliminate diff caused by reordering.

The diff algorithm itself is Set-based, so there it does not matter. But it does matter in keeping git diff small in cases like yours :).

Reply:
Probably because since our last change to surface area test generator (https://github.com/dotnet/fsharp/pull/11574), we haven't re-generated them.
We should add sorting to it, so it's deterministic order every time.

Reply:
> @nojaf : I assume you just copy-pasted 'actual' into this expected string, right?

Yes sir, guilty 😅

I'll try the sorting.